README.md

```diff
@@ -30,6 +30,7 @@
 ## Readme Generator
 
 - [Get Readme](https://github.com/luctst/get-readme#readme) - CLI tool to generate an esthetic readme file 🔖.
+- [Plantek README Manager](https://gitlab.com/MarleyPlant/readme-manager) - Stop wasting time on READMEs. This CLI tool generates professional documents for your Git projects with ease.
```

Comment:
@MarleyPlant you forgot to move it after `GitHub Profilinator` according to alphabetical order.